### DISCUSSION TITLE
XG mobile

### DISCUSSION THREAD

**Discussion by craterboy1** [seerge/g-helper#2891] [Ideas]
Hi Serge, 

Great work on this! Could we get an automatic xgmobile connect option where it automatically engages the xgmobile upon detection. That way I dont have to fumble through settings to use it on my main monitor every time

Regards

**Comment by seerge** [comment]
@craterboy1 hello, G-Helper is not monitoring connection / disconnection of system devices in realtime. It updates XGM UI on the main window only when you re-open it. So I don't see a big problem in clicking XGM button once :)

**Comment by craterboy1** [comment]
Hi,

Thanks for the response!  I guess the main purpose would be to plug my laptop into my main monitor with fumbling with my laptop since the main display will not activate unless xgmobile is toggled. But, I understand this may be a niche case and not useful for a lot of people

Thanks anyways
